lib/promrelabel/if_expression.go

```diff
@@ -40,6 +45,18 @@ func (ie *IfExpression) Match(labels []prompbmarshal.Label) bool {
 	return false
 }
 
+// ToLabelValue returns label value created from IfExpression
+func (ie *IfExpression) ToLabelValue() string {
```

Comment:
I don't think this function should be here due to its `maxLabelValueLen ` limit, which doesn't seem general but rather specific to what we're doing in streamaggr lib.

Reply:
moved logic to lib/streamaggr